megazords/nimbus-ios/NimbusStandalone.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,587 @@
+// !$*UTF8*$!
```

Comment:
This and its supporting `.swift` files were copied from the iOS megazord and heavily trimmed down. Depending on how the build process shapes up, we might want to consider moving e.g. the Swift RustLog and Viaduct stuff out of the iOS megazord and into some shared location.